util/uvmdvgen/tb.sv.tpl

```diff
@@ -41,6 +34,10 @@ module tb;
   ${agent}_if ${agent}_if();
 % endfor
 
+% if has_alerts:
+  `DV_ALERT_IF_CONNECT
+% endif
+
```

Comment:
I think line 62 to 73 can also be deleted with the macro.

Reply:
Updated, thanks